doc/user/content/get-started/key-concepts.md

```diff
@@ -23,13 +23,47 @@ Materialize offers the following components through its SQL API:
 
 Component                | Use
 -------------------------|-----
+**[Clusters]**           | Isolated pools of compute resources for sources, sinks, indexes, materialized views, and queries.
 **[Sources]**            | Sources describe an external system you want Materialize to read data from (e.g. Kafka).
 **[Views]**              | Views represent queries of sources and other views that you want to save for repeated execution.
 **[Indexes]**            | Indexes represent query results stored in memory.
 **[Materialized views]** | Materialized views represent query results stored durably.
-**[Sinks]**              | Sinks represent output streams or files that Materialize sends data to.
-**[Clusters]**           | Clusters describe logical compute resources that can be used by sources, sinks, indexes, and materialized views.
-**[Cluster replicas]**   | Cluster replicas allocate physical compute resources for a cluster.
+**[Sinks]**              | Sinks represent output streams that Materialize sends data to.
+
+## Clusters
+
+Clusters are pools of compute resources (CPU, memory, and, optionally, scratch
+disk space) for running your workloads.
+
+The following operations require compute resources in Materialize, and so need
+to be associated with a cluster:
+
+- Executing [`SELECT`] and [`SUBSCRIBE`] statements.
```

Comment:
off-topic, I wonder if subscribe should be a key concept.

Reply:
Would make sense!